### PR TITLE
qemu and parallels patterns in ledmanager

### PR DESCRIPTION
**Pull request by giggsoff**
Check parallels and qemu by patterns inside ledmanager

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: pkg/pillar/cmd/ledmanager/ledmanager.go [hunk 1]
```diff
@@ -204,25 +209,36 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 	ps.StillRunning(agentName, warningTime, errorTime)
 
 	model := hardware.GetHardwareModel(log)
-	log.Functionf("Got HardwareModel %s", model)
+	log.Warningf("Got HardwareModel %s", model)
```
Comment (eriknordmark) [inline pkg/pillar/cmd/ledmanager/ledmanager.go:212]: Why a warning? Log.Noticef should suffice to have it be sent to the controller log API by default.

**Review by eriknordmark — APPROVED** [review]
Just a nit about log.Warning vs log.Function

**Review by rvs — APPROVED** [review]
LGTM (sans what @eriknordmark said)

File: pkg/pillar/cmd/ledmanager/ledmanager.go (hunk 1 above)
Comment (giggsoff) [reply pkg/pillar/cmd/ledmanager/ledmanager.go]: Corrected, thanks

**Pull request merged**